userguide/advanceduse/ssh.rst

```diff
@@ -50,6 +50,13 @@ Connect
 Now everything is set up and you can use ``ssh`` ::
 
     ssh phablet@<ip-address>
+    
+To identify the ip-address of your UT device , open Terminal application in your phone and run the following command::
+    
+    hostname -I
+    
+The output is a list of IP addresses seperated by spaces.Use the first IP address from the list.
```

Comment:
I only notice now, sorry - it's "separated" - second vowel is an a not e. 

Space before "Use" please.

Reply:
Made typo correction & addeded the space before use